pyproject.toml

```diff
@@ -12,9 +12,11 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.9.1,<3.10"
 opentimelineio = "^0.16.0"
-pytest = "^6.2.0"
 opencolorio = "^2.3.2"
-selenium = "^4.20.0"
+selenium = "4.16.0"
+
+[tool.poetry.dev-dependencies]
```

Comment:
this is cool!
how would poetry know whether it's being installed for dev purposes or not?
don't know poetry that well.... yet ;)

Reply:
oh cool! that's very handy!
thanks for pointing out :)